src/week03/gahyun/p2/Main.java

```diff
@@ -0,0 +1,31 @@
+package week03.gahyun.p2;
+
+/**
+ * 문제 링크 : https://school.programmers.co.kr/learn/courses/30/lessons/43165
+ * 메모리 : 	75.2MB
+ * 시간 : 4.52ms
+ */
+
+/**
+ * DFS
+ * 값 +- 여부로 분기처리되는 트리 형태 (->전위순회)
+ * 각 레벨 진행될 때마다 배열값 더하거나 뺌
+ * 리프 노드에서 총합이 타겟 넘버면 카운트 증가
+ */
+class Solution {
+	static int answer = 0;
+	public int solution(int[] numbers, int target) {
+		DFS(0, 0, numbers, target);
+		return answer;
+	}
+	public static void DFS(int lv, int total, int[] arr, int target){
+		if (lv==arr.length){
+			if (total==target) {
+				answer++;
+			}
+		} else {
+			DFS(lv+1, total+arr[lv], arr, target);
+			DFS(lv+1, total-arr[lv], arr, target);
+		}
```

Comment:
```suggestion
			}
			 return;
		}
		DFS(lv+1, total+arr[lv], arr, target);
		DFS(lv+1, total-arr[lv], arr, target);
```
이건 취향 차이인데, 이렇게 하는 방법도 있어요! 😄

Reply:
오호 else로 안빼고 바로 return 할 수도 있었네요! 감사합니당